test/wasm_worker/terminate_wasm_worker.c

```diff
@@ -45,11 +45,15 @@ char stack[1024];
 int should_throw(void(*func)())
 {
   int threw = EM_ASM_INT({
+    var oldAbort = abort;
+    abort = () => { throw 'abort' };
```

Comment:
Is the test change related?

Reply:
This test is trying to try/catch around the use of an invalid handle.  However, in the handle allocate we `assert` on invalid handle usage, which then calls `abort`.   Sadly `abort` is not something that can be caught in the normal try/catch because it brings down the runtime before throwing, so you can't reasonably recover from it.

Perhaps you should be able to.. but perhaps not.   Once a program calls `abort` I'm not sure we want it to be able to continue...  so this test now has to a slightly more horrible hack of overriding the abort call itself.